mkdocs.yml

```diff
@@ -34,6 +35,8 @@ nav:
         - Configuring Nautobot: 'configuration/index.md'
         - Required Settings: 'configuration/required-settings.md'
         - Optional Settings: 'configuration/optional-settings.md'
+    - User Guides:
+        - Git as a Data Source: 'user-guides/git-data-source.md'
```

Comment:
One last change request: can you move this section down by one, so that it falls between `External Authentication` and `Core Functionality`?

Reply:
@glennmatthews done.